src/jsutils/instanceOf.ts

```diff
@@ -1,5 +1,14 @@
 import { inspect } from './inspect';
 
+/* c8 ignore next 4 */
+const isProduction =
+  // eslint-disable-next-line no-undef
+  typeof process !== 'undefined' &&
+  // eslint-disable-next-line no-undef
+  process.env &&
+  // eslint-disable-next-line no-undef
+  process.env.NODE_ENV === 'production';
```

Comment:
Another alternative which does not fix the `id="process"` bug

```suggestion
/* c8 ignore next 3 */
const isProduction =
  // eslint-disable-next-line no-undef
  typeof process !== 'undefined' &&
  // eslint-disable-next-line no-undef
  process.env.NODE_ENV === 'production';
```